diff --git a/docker/build_artifacts/sagemaker/python_service.py b/docker/build_artifacts/sagemaker/python_service.py
index e294e5f..05444ec 100644
--- a/docker/build_artifacts/sagemaker/python_service.py
+++ b/docker/build_artifacts/sagemaker/python_service.py
@@ -238,6 +238,16 @@ class PythonServiceResource:
                     data, context = tfs_utils.parse_request(req, rest_port, grpc_port,
                                                             self._tfs_default_model_name,
                                                             model_name=model_name)
+                    try:
+                        res.status = falcon.HTTP_200
+
+                        res.body, res.content_type = self._handlers(data, context)
+                    except Exception as e:  # pylint: disable=broad-except
+                        log.exception("exception handling request: {}".format(e))
+                        res.status = falcon.HTTP_500
+                        res.body = json.dumps({
+                            "error": str(e)
+                        }).encode("utf-8")  # pylint: disable=E1101
             else:
                 res.status = falcon.HTTP_400
                 res.body = json.dumps({
@@ -251,16 +261,16 @@ class PythonServiceResource:
                                                     self._tfs_default_model_name,
                                                     channel=self._channels[grpc_port])
 
-        try:
-            res.status = falcon.HTTP_200
+            try:
+                res.status = falcon.HTTP_200
 
-            res.body, res.content_type = self._handlers(data, context)
-        except Exception as e:  # pylint: disable=broad-except
-            log.exception("exception handling request: {}".format(e))
-            res.status = falcon.HTTP_500
-            res.body = json.dumps({
-                "error": str(e)
-            }).encode("utf-8")  # pylint: disable=E1101
+                res.body, res.content_type = self._handlers(data, context)
+            except Exception as e:  # pylint: disable=broad-except
+                log.exception("exception handling request: {}".format(e))
+                res.status = falcon.HTTP_500
+                res.body = json.dumps({
+                    "error": str(e)
+                }).encode("utf-8")  # pylint: disable=E1101
 
     def _setup_channel(self, grpc_port):
         if grpc_port not in self._channels:
